src/components/LowerLayout/Footer/Footer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import OpenUpLogo from '../../../images/OpenUp-logo.png'
+
+import global from '../../../styles.module.css';
+import classes from './Footer.module.css';
+
+const Footer = () => {
+  return (
+    <div>
+      <div className={global.containerWithUnderline}>
+        <img className={classes.logo} src={OpenUpLogo} alt=""/>
+      </div>
+      <div className={global.container}>
+        <div className={classes.CollectionDateAndContact}>
```

Comment:
Once again, I would write this as camel case, otherwise it might seem like they are JavaScript class constructors (not to be confused with CSS classes). 

https://stackoverflow.com/questions/1008230/what-is-the-best-practice-casing-style-for-javascript-why